### DISCUSSION TITLE
Github Issue Development 관련 질문

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#108] [Q&A] [answered]
[문제가 되는 이슈 링크](https://github.com/BRIDGE-DEV/BRIDGE_BookClub/issues/62)

스터디원이 실수로 생성한 브랜치를 삭제했지만 바인딩된 이슈에서 삭제가 되지 않아서 글을 올립니다.

구글링헤도 정보가 나오지 않아서 혹시나 아실까 해서..

![image](https://github.com/fkdl0048/Mentoring-Project/assets/84510455/e3729f5d-4a00-4ea1-b666-0049fe769688)

- 생성된 브랜치를 삭제하였지만 사라지지 않고, 실제로 접근해보면 error라고 뜹니다..
![image](https://github.com/fkdl0048/Mentoring-Project/assets/84510455/61b8df26-bf37-4683-ac0e-682c66c4ec6f)

**Comment by jongfeel** [comment, marked as answer]
이건 지난 번 오프라인 때 잠깐 확인했던 건데, 아직까지는 어쩔 수 없는 기능인 것 같습니다.
논의했던 대로 github community에 제보하면 언젠가는 수정해 줄거 같긴 합니다.

**Reply by fkdl0048** [reply]
https://github.com/orgs/community/discussions/75846

4일 전 쯤에 올렸는데 언젠간 해결이 될 것 같습니다 ㅎ!
